src/leveled_cdb.erl

```diff
@@ -424,6 +432,20 @@ cdb_isrolling(Pid) ->
 cdb_clerkcomplete(Pid) ->
     gen_fsm:send_all_state_event(Pid, clerk_complete).
 
+-spec cdb_getcachedscore(pid(), erlang:timestamp()) -> undefined|float().
+%% @doc
+%% Return the cached score for a CDB file
+cdb_getcachedscore(Pid, Now) ->
```

Comment:
why not computing the timestamp inside the call? A bit weird to have it passed in through the interface, but probably that's for testing...

```
cdb_getcachedscore(Pid) ->
    gen_fsm:sync_send_all_state_event(Pid, {get_cachedscore, erlang:timestamp()}, infinity).
```